tools/tools.go

```diff
@@ -135,3 +136,23 @@ func BuildCidFromMessageTrace(msg filTypes.MessageTrace, parentMsgCid string) (s
 
 	return b.Cid().String(), nil
 }
+
+func SetNodeMetadataOnTxs(txs []*types.Transaction, metadata types.BlockMetadata, parserVer string) []*types.Transaction {
+	nodeMajorMinorVersion := metadata.NodeMajorMinorVersion
+	if nodeMajorMinorVersion == "" {
+		nodeMajorMinorVersion = "unknown"
```

Comment:
maybe print an error msg here? it 'should' not happen, though

Reply:
It will generate a huge amount of logs... not sure if it is really necessary, as it is not super important